server/app/assets/javascripts/admin_programs.ts

```diff
@@ -4,6 +4,21 @@ class AdminPrograms {
   private static PROGRAM_CARDS_SELECTOR = '.cf-admin-program-card'
   private static PROGRAM_LINK_ATTRIBUTE = 'data-copyable-program-link'
 
+  static attachConfirmCommonIntakeChangeListener() {
+    document
+      .querySelector('#confirm-common-intake-change-button')
+      ?.addEventListener('click', () => {
+        const confirmationCheckbox = <HTMLInputElement>(
+          document.querySelector('#confirmed-change-common-intake-checkbox')
+        )
+        if (!confirmationCheckbox) {
+          return
+        }
+        confirmationCheckbox.value = 'true'
+        confirmationCheckbox.checked = true
+      })
+  }
```

Comment:
**suggestion (non-blocking):** consider using the shared function instead: https://github.com/civiform/civiform/blob/main/server/app/assets/javascripts/util.ts#L14

Reply:
Done